<!DOCTYPE html>
<html lang="en">
  <head>
    <title>SDA - Blog </title>
    <meta charset="utf-8">
    <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
    
    <link href="https://fonts.googleapis.com/css?family=Poppins:300,400,500,600,700" rel="stylesheet">
    <link href="https://fonts.googleapis.com/css?family=Montserrat:300,400,500,700" rel="stylesheet">

    <link rel="stylesheet" href="../css/open-iconic-bootstrap.min.css">
    <link rel="stylesheet" href="../css/animate.css">
    
    <link rel="stylesheet" href="../css/owl.carousel.min.css">
    <link rel="stylesheet" href="../css/owl.theme.default.min.css">
    <link rel="stylesheet" href="../css/magnific-popup.css">

    <link rel="stylesheet" href="../css/aos.css">

    <link rel="stylesheet" href="../css/ionicons.min.css">

    <link rel="stylesheet" href="../css/bootstrap-datepicker.css">
    <link rel="stylesheet" href="../css/jquery.timepicker.css">

    
    <link rel="stylesheet" href="../css/flaticon.css">
    <link rel="stylesheet" href="../css/icomoon.css">
    <link rel="stylesheet" href="../css/style.css">

    <link rel="icon" type="image/png"  href="../SDA.png">

  </head>
  <body>

	<div id="colorlib-page">
		<a href="#" class="js-colorlib-nav-toggle colorlib-nav-toggle"><i></i></a>
		<aside id="colorlib-aside" role="complementary" class="js-fullheight text-center">
			<h1 id="colorlib-logo"><a href="../index.html">SDA<span>.</span></a></h1>
			<nav id="colorlib-main-menu" role="navigation">
        <ul>
					<li><a href="index.html">Home</a></li>
					<li class="colorlib-active"><a href="blog.html">Blog</a></li>
          <li><a href="projects.html">Projects</a></li>
          <li><a href="training.html">Training</a></li>

					<!-- <li><a href="photography.html">Photography</a></li> -->
					<!-- <li><a href="fashion.html">Fashion</a></li> -->
					<li><a href="contact.html">Contact</a></li>
				</ul>
			</nav>

			<div class="colorlib-footer">
        <ul>
					<li><a href="https://github.com/genomeorganizer"><i class="icon-github"></i></a></li>
					<li><a href="https://www.linkedin.com/in/sdayers/"><i class="icon-linkedin"></i></a></li>
					<li><a href="https://twitter.com/genomeorganizer"><i class="icon-twitter"></i></a></li>
				</ul>
			</div>
		</aside> <!-- END COLORLIB-ASIDE -->
		<div id="colorlib-main">
			<!-- <div class="hero-wrap hero-wrap-2 js-fullheight" style="background-image: url(../images/bg_2.jpg);" data-stellar-background-ratio="0.5"> -->
            <div class="hero-wrap hero-wrap-2" data-stellar-background-ratio="0.5">

                <!-- <div class="overlay"></div> -->
				<div class="d-flex justify-content-center align-items-center">
					<div class="col-md-8 text text-center">
						<!-- <div class="img mb-4" style="background-image: url(../images/SDA_Busy.gif);"></div> -->
						<div class="desc">
							<!-- <h2 class="subheading">Hello I'm</h2> -->
                            <h3 class="mb-2">What's Next</h3>
                            <h2 class="subheading">4/19/2014</h2>
                            <ul class="ftco-social mt-3">  
                                <li class="ftco-animate"><a href="https://github.com/genomeorganizer"><i class="icon-github"></i></a></li>
                                <li class="ftco-animate"><a href="https://www.linkedin.com/in/sdayers/"><i class="icon-linkedin"></i></a></li>
                                <li class="ftco-animate"><a href="https://twitter.com/genomeorganizer"><i class="icon-twitter"></i></a></li>
                            </ul>
						</div>
                    </div>

                </div>


          <div class="text-left font-weight-normal" style="color:black">
            <div dir="ltr" style="text-align: left;" trbidi="on">
                Almost as soon as the first genome was sequenced at the start of the last decade, the race was on to bring whole genome sequencing to the consumer level. As we look back over this time, we see the remnants of this contest: 454, Helicos and others, until now Illumina seems to be the winner of this round with a few other stragglers left behind, as we approach the long sought-after goal of the $1,000 genome. As we reach this point, genome sequencing will undergo an important transition from a headline-grabbing research methodology to a practical tool for clinical diagnosis. Clinical applications will bring a new influx of funding, as it is applied on a wide scale and as healthcare providers inflate the price of the $1,000 genome to $10,000 and more (source: my street smarts). Clinical applications will impose unique market demands on this technology, and it makes sense for us now to ask what these new demands will mean for the NGS industry.<br />
                <br />
                <b>Size Matters</b><br />
                Until now, 'Short Read" sequencing has been synonymous with NGS, reflecting the fact that most of the early NGS technologies yielded  high base output by producing large numbers of short reads, ranging from 10s to 100s of bases, in contrast to hitherto standard Sanger sequencing, which was capable of reading sequences of more than 1,000 bases. As demonstrated by the progress of research results over the past decade, a wide range of information can be gleaned from such short reads, but they do have their limitations. As <i>de novo</i> genome assembly becomes a more common application in clinical settings, there is a greater need for sequencing techniques that can get past these short read lengths to sequence through highly repetitive regions of the genome, which are often just ignored in current sequencing approaches as "unmappable." While the read length of 454 eventually grew to over 2,000 bases (even if its total base output remained too low to be usable), modern sequencing methodologies have had a hard time increasing their read lengths. Illumina's sequencing platforms are currently limited to 150bp 
                paired-end reads, while Ion Torrent sequencing is limited to 200bp. While the total base output of these platforms is enough to hold the public's imagination for now, it seems only a matter of time until they are forced to come to terms with the shortcomings of these sequencing approaches.<br />
                <br />
                <b>Limits of Clonal Methods</b><br />
                <b></b><br />
                Despite the diversity of sequencing approaches, most (including Illumina, Ion Torrent, Complete Genomics, SOLID) have in common the use of clonal amplification to increase signal strength--libraries are ligated to various sorts adapters and amplified by PCR. The resulting products are attached to flowcells and sequencing progress (polymerase or ligase-based) monitored by imaging of fluorescently-labeled base analogues. These type of approaches are inherently limited in their ability to extend beyond a few hundred bases, since errors are cumulative, causing the error rate to increase exponentially with read length. After a few hundred bases, it's no longer possible to keep reads in phase within a given colony at a suitably high quality level.<br />
                <br />
                <b>Size Enhancement</b><br />
                Recognizing these limitations, NGS developers have sought to augment read length through enhancement of their library processing protocols. Illumina recently acquired Moleculo, which had developed a method to extend read length through the interspersal of adapter sequences at intermediate points within the libraries and they have claimed reads of over 10kb through this process. Complete Genomics has taken an alternative approach, LFR, which segregates genomic DNA into smaller pools that are independently sequenced. Similar in principle to older BAC-based sequencing of genomes, this technique is also capable of producing large continuous sequences.<br />
                <br />
                <b>Remember Single Molecule?</b><br />
                In principle, single molecule sequencing strategies (of which only Pacific Biosciences remains) offer a simple alternative; since each read is obtained from one DNA molecule, there is no way for the read to go out of phase. PacBio's read length is now upwards of 20,000 bases. Although PacBio's single read quality scores have never gotten north of 75%, they can push this score up to within acceptable Q30 ranges through aggregation of multiple reads. Although its total read output has never increased above 100,000, experience has shown read count to be easier to increase than read length. Given these observations, we can't deny that PacBio might have something to bring to the current battle of sequencing platforms.<br />
                <br />
                <b>The Biggest Stick</b><br />
                Based on these observations, it seems that we may be in for a new kind of race between sequencing platforms. Although Illumina is at the head out of the gate, it's too early to discount dark horses like PacBio. As applications shift from research to the clinic, needs will also change and time will reveal the next winner.</div>

                
                
            </div>


      </div>
      
      <footer class="ftco-footer" style="color: black;background-color: whitesmoke;">
        <p style="color:grey;"> Copyright &copy;<script>document.write(new Date().getFullYear());</script> All rights reserved | This template is made with <i class="icon-heart" aria-hidden="true"></i> by <a href="https://colorlib.com" target="_blank" style="color:black">Colorlib</a></p>
        </footer>
        
		</div><!-- END COLORLIB-MAIN -->
	</div><!-- END COLORLIB-PAGE -->

  <!-- loader -->
  <div id="ftco-loader" class="show fullscreen"><svg class="circular" width="48px" height="48px"><circle class="path-bg" cx="24" cy="24" r="22" fill="none" stroke-width="4" stroke="#eeeeee"/><circle class="path" cx="24" cy="24" r="22" fill="none" stroke-width="4" stroke-miterlimit="10" stroke="#F96D00"/></svg></div>


  <script src="../js/jquery.min.js"></script>
  <script src="../js/jquery-migrate-3.0.1.min.js"></script>
  <script src="../js/popper.min.js"></script>
  <script src="../js/bootstrap.min.js"></script>
  <script src="../js/jquery.easing.1.3.js"></script>
  <script src="../js/jquery.waypoints.min.js"></script>
  <script src="../js/jquery.stellar.min.js"></script>
  <script src="../js/owl.carousel.min.js"></script>
  <script src="../js/jquery.magnific-popup.min.js"></script>
  <script src="../js/aos.js"></script>
  <script src="../js/jquery.animateNumber.min.js"></script>
  <script src="../js/bootstrap-datepicker.js"></script>
  <script src="../js/jquery.timepicker.min.js"></script>
  <script src="../js/scrollax.min.js"></script>
  <script src="../https://maps.googleapis.com/maps/api/js?key=AIzaSyBVWaKrjvy3MaE7SQ74_uJiULgl1JY0H2s&sensor=false"></script>
  <script src="../js/google-map.js"></script>
  <script src="../js/main.js"></script>
    
  </body>
</html>